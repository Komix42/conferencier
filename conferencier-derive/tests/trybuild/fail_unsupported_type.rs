use std::collections::HashMap;

#[derive(conferencier_derive::ConferModule)]
struct BadType {
    map: HashMap<String, String>,
}

fn main() {}
