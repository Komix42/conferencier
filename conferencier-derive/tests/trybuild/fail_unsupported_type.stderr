error: unsupported field type `HashMap`
 --> tests/trybuild/fail_unsupported_type.rs:5:10
  |
5 |     map: HashMap<String, String>,
  |          ^^^^^^^
